modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -290,6 +290,11 @@ The API icon, name, and description shown are set in the API Community Manager c
 You can specify values for the following parameters of the API Header:
 
 API:: Specifies the API to show.
```

Comment:
Just curious about the double :: Is this a style thing I'm not aware of? Most of the docs use just a single : to separate a term and its definition, for example.

Reply:
The double colon makes the list into a definition list with slightly different style and formatting; you can see them here: https://docs.mulesoft.com/api-community-manager/acm-lightning-components